### ISSUE TITLE
Sync locale with child processes on Windows

### ISSUE THREAD

**Issue by matyalatte** [matyalatte/tuw#36]
Related to #34.
A typical nightmare due to Windows locale.
Tuw can show garbled strings if a child process does not use the default locale.
I should find a way to get locale for child processes or provide an option to set it by users.

I think there is nothing to do for unix-like systems because they use utf-8 basically.

**Comment by matyalatte** [comment]
Supporting all locales would be hard to test, so I decided to support utf-8 only.
I'll add an option `codepage`.

```json
{
    "label": "UTF-8 output",
    "command": "a command that outputs utf-8 strings",
    "codepage": "utf-8",
    "components": []
 }
```

When `utf-8` or `utf8`, tuw converts utf-8 strings to wstrings on Windows.
When `default`, tuw redirects output strings as it is.
Non-Windows platforms ignore the option.

**Comment by matyalatte** [comment]
Fixed with #43.
